app/components/support_interface/application_summary_component.rb

```diff
@@ -178,6 +180,47 @@ def subscribed_to_emails?
       candidate.subscribed_to_emails?
     end
 
+    def find_a_candidate_state_row
+      {
+        key: 'Find a Candidate opt-in status',
+        value: if application_form_in_the_pool?
+                 'Currently in the candidate pool'
+               elsif candidate.published_opt_in_preferences.present?
+                 'Opted in (not in pool)'
+               elsif candidate.published_preferences.last&.opt_out?
+                 'Opted Out'
+               else
+                 'No status recorded'
+               end,
+      }
+    end
+
+    def find_a_candidate_location_preferences_row
+      return if candidate.published_opt_in_preferences.blank?
+
+      location_preferences = candidate.published_opt_in_location_preferences
+      decorated_preferences = location_preferences.map { |location| LocationPreferenceDecorator.new(location) }
+
+      value =
+        govuk_list do
+          decorated_preferences.map do |location|
+            tag.li t('.location', radius: location.within, location: location.decorated_name)
+          end.join.html_safe
+        end
+
+      {
+        key: 'Find a Candidate location preferences',
+        value: location_preferences.any? ? value : 'No location preferences recorded',
+      }
+    end
+
+    def application_form_in_the_pool?
+      return false if candidate.submission_blocked? || candidate.account_locked?
+      return false if candidate.published_opt_in_preferences.blank?
+
+      Pool::Candidates.new(providers: []).application_forms_in_the_pool.exists?(id: application_form.id)
```

Comment:
This is a pretty heavy query -- I wonder if there are individual checks on a candidate that can be performed rather than getting all the applications first? I might be overly worried though, about how long this query will take. it might be worth all of us looking at it together if @dcyoung-dev and @CatalinVoineag agree.